<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMenu</class>
 <widget class="QDialog" name="AdminMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/adminmenu.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_user_submit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>231</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_change_user">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_change_pass">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pass_submit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_delete_user">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit_delete">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>395</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search_user">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit_search">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_lifetime">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>12</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>List of Lifetime Users</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_change_user">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Change Username</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_change_pass">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>12</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>12</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Add More Fortunes</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>300</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>12</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Search User</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>300</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ageo</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_user_submit</sender>
   <signal>clicked()</signal>
   <receiver>AdminMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pass_submit</sender>
   <signal>clicked()</signal>
   <receiver>AdminMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
